### DISCUSSION TITLE
Mark article as read/done

### DISCUSSION THREAD

**Discussion by Trubador** [dailydotdev/daily#1333] [Feature Requests]
I would like to be able to mark an article as having been read and done so it does not show in my feed or bookmarks when I am done reading it

**Comment by idoshamun** [comment]
To remove something from your bookmarks, simply unbookmark it.
If you want to remove something from your feed, you can use the hide feature.
Does that work for you?

**Reply by Trubador** [reply]
Yes it does though I would prefer sending the stories and booksmarks I have read to an archive list to be able to look them up in the future in case I want to read them again for additional insights.
